config/environments/development.rb

```diff
@@ -66,4 +66,6 @@
     Rails.application.credentials.author_username.presence
   config.x.author_password = ENV["AUTHOR_PASSWORD"].presence || \
     Rails.application.credentials.author_password.presence
+
+  config.x.api_client_cache_store = ActiveSupport::Cache::MemoryStore.new
```

Comment:
This is causing a spec to fail due to testing for `cache_read` on active support but its never called now that we've isolated this to a separate cache.